doc/cephfs/client-auth.rst

```diff
@@ -208,60 +208,104 @@ such as ``ceph health detail``.
 MDS communication restriction
 =============================
 
-By default, user applications may communicate with any MDS, whether or not
-they are allowed to modify data on an associated file system (see
-`Path restriction` above). Client's communication can be restricted to MDS
-daemons associated with particular file system(s) by adding MDS caps for that
+By default, user applications may communicate with any MDS, regardless of
+whether they are allowed to modify data on an associated file system (see `Path
+restriction` above). Client communication can be restricted to MDS daemons
+associated with particular file system(s) by adding MDS caps for that
 particular file system. Consider the following example where the Ceph cluster
-has 2 FSs::
+has two file systems:
+
+.. prompt:: bash #
+
+   ceph fs ls
+
+::
 
-    $ ceph fs ls
     name: cephfs, metadata pool: cephfs_metadata, data pools: [cephfs_data ]
     name: cephfs2, metadata pool: cephfs2_metadata, data pools: [cephfs2_data ]
 
-Client ``someuser`` is authorized only for one FS::
+Client ``someuser`` is authorized for only one file system:
+
+.. prompt:: bash #
+
+   ceph fs authorize cephfs client.someuser / rw
+
+::
 
-    $ ceph fs authorize cephfs client.someuser / rw
     [client.someuser]
         key = AQBPSARfg8hCJRAAEegIxjlm7VkHuiuntm6wsA==
-    $ ceph auth get client.someuser > ceph.client.someuser.keyring
+
+.. prompt:: bash #
+
+   ceph auth get client.someuser > ceph.client.someuser.keyring
+
+::
+
     exported keyring for client.someuser
-    $ cat ceph.client.someuser.keyring
+
+.. prompt:: bash #
+
+   cat ceph.client.someuser.keyring
+
+::
+
     [client.someuser]
         key = AQBPSARfg8hCJRAAEegIxjlm7VkHuiuntm6wsA==
         caps mds = "allow rw fsname=cephfs"
         caps mon = "allow r"
         caps osd = "allow rw tag cephfs data=cephfs"
 
-Mounting ``cephfs1`` with ``someuser`` works::
+Mounting ``cephfs1`` with ``someuser`` works:
+
+.. prompt:: bash #
+
+   sudo ceph-fuse /mnt/cephfs1 -n client.someuser -k ceph.client.someuser.keyring --client-fs=cephfs
```

Comment:
In these examples, does one need to mkdir the mountpoint first?  With NFS one typically does (unless you're DomainOS, where it wouldn't work if the mountpoint existed)

Reply:
I'll pick this up in a future PR.

Reply:
https://github.com/ceph/ceph/pull/56879